app/views/articles/new.html.haml

```diff
@@ -1,4 +1,4 @@
-- if !@article.valid?
+- if !@article.valid? && !@article.blank?
```

Comment:
Is this change really necessary?

https://github.com/orientation/orientation/blob/master/app/controllers/articles_controller.rb#L23-L25